### ISSUE TITLE
Get child process message

### ISSUE THREAD

**Issue by pincman** [remy/nodemon#1870]
How to get child process message,like server run callback?
When run a nestjs server,like follow
```typescript
const runner = nodemon({
        script: 'src/main.ts',
        ext: 'js,json,ts',
        watch: ['src'],
        ignore: ['.git', 'node_modules', 'dist', 'scripts'],
        execMap: {
            ts: 'eslint "{src,apps,libs,test}/**/*.ts" --fix && node -r @swc/register -r module-alias/register',
        },
        env: {
            NODE_ENV: environment(),
        },
        cwd: path.resolve(__dirname, '..', '..'),
        spawn: true,
    });
    runner.once(
        'start',
        () => {
            ora('Server is in starting...').start();
        },
    );
// get subprocess message to stop spinner
```
main.ts
```typescript
await app.listen(appConfig.port, appConfig.host, () => {
        console.log();
        console.log('Server has started:');
        const listens: string[] = [];
        const nets = networkInterfaces();
// Here to send server started message
        Object.entries(nets).forEach(([_, net]) => {
            if (net) {
                for (const item of net) {
                    if (item.family === 'IPv4') listens.push(item.address);
                }
            }
        });
        const urls = listens.map(
            (l) =>
                `${appConfig.https ? 'https' : 'http'}://${l}:${
                    appConfig.port
                }`,
        );
        if (urls.length > 0) {
            console.log(`- Local: ${chalk.green.underline(urls[0])}`);
        }
        if (urls.length > 1) {
            console.log(`- Network: ${chalk.green.underline(urls[1])}`);
        }
    });
```
and I want to stop `ora` spinner after server started

**Comment by remy** [comment]
You want to hook the `log` or `stdout` events and then parse for when your subprocess has finished it's bootup: https://github.com/remy/nodemon/wiki/Events#messages

**Comment by pincman** [comment]
@remy 
thx for your reply
The `log` or `stdout` event is only triggered every time the subprocess outputs, and it is impossible to get the send data by any time of the subprocess, like a fork.
I tried to use the `process.send` function in the subprocess `main.ts`, but the function is  `undefined`, so does nodemon use fork to start the subprocess? And I have set this in the configuration: `spawn: false`,but process.send is always `undefined`
the code like follow
```typescript
// main.ts
await app.listen(appConfig.port, appConfig.host, () => {
process.send('started')
...
```
cli with nodemon
```typescript
// cli.ts
const runner = nodemon(...)
runner.on('message',(m) => {
    if(m === 'started') ...
})
```
so how to solve it or reopen this issue? thanks again
